### ISSUE TITLE
Allow unit values in enum validation

### ISSUE THREAD

**Issue by JohannesIBK** [jprochazk/garde#74]
Currently you cannot validate enums that contain unit-like fields without implementing the trait yourself. Is it possible to automatically skip fields like this, or at least make it respect `#[garde(skip)]`?

Code:

```rust
#[derive(garde::Validate)]
pub enum Test {
    Unit,
    Tuple(#[garde(range(min = 1, max = 10))] i32),
    Struct {
        #[garde(range(min = 1, max = 10))]
        value: i32,
    },
}
```

**Comment by jprochazk** [comment]
That's definitely a bug. The proc macro doesn't generate a match arm, even though it should generate an empty one.

**Comment by jprochazk** [comment]
Published in [v0.16.2](https://github.com/jprochazk/garde/releases/tag/v0.16.2)

**Comment by JohannesIBK** [comment]
Wow, this was fast, thank you very much
